src/content/docs/assist/index.mdx

```diff
@@ -5,29 +5,31 @@ description: Learn about Biome assist
 
 import NumberOfRules from "@/components/generated/assist/NumberOfRules.astro";
 import PackageManagerBiomeCommand from "@/components/PackageManagerBiomeCommand.astro";
+import EditorAction from "@/components/EditorAction.astro";
 
-Biome assist offers a series of actions (or rules) meant to improve code quality and DX of the users.
+Biome assist offers a series of actions meant to improve code quality and DX of the users.
```

Comment:
If you use it as a proper noun, I would suggest capitalising both words in "Biome Assist". Otherwise, "The Biome assist" or "Biome's assist" would be more appropriate.

```suggestion
Biome Assist offers a series of actions meant to improve code quality and DX of the users.
```